base/constants.go

```diff
@@ -189,6 +189,27 @@ var (
 	ErrUnknownField = errors.New("unrecognized JSON field")
 )
 
+func DCPCheckpointPrefixWithGroupID(groupID string) string {
+	if groupID != "" {
+		return DCPCheckpointPrefix + groupID + ":"
+	}
+	return DCPCheckpointPrefix
+}
+
+func SGCfgPrefixWithGroupID(groupID string) string {
+	if groupID != "" {
+		return SyncPrefix + groupID + ":cfg:"
```

Comment:
Previously we didn't have a trailing colon on SGCfgPrefix.  This doesn't look backward compatible if the groupID is empty?

Reply:
If group ID is empty, the old `SGCfgPrefix` is returned, which is now a separate constant (see return below).

Reply:
This format - `_sync:groupID:cfg:` doesn't match the discussion on the JIRA issue, which is: 

> Scope CBGT config by group ID as you have - (i.e. _sync:cfg:group1:foobar and fall back to _sync:cfgfoobar for default group)

If we followed the approach on the ticket, we wouldn't need to be passing the SGCfg prefix down to DCPCommon, and I think that means we could also remove it from FeedArgs.  If there was a reason to use the format you've got here, can you provide the details?